[7.48.x] DROOLS-5788: [DMN Designer] Show parameters list for Decision Services - Release notes (#3133)
kiegroup/kie-docs: doc-content/shared-kie-docs/src/main/asciidoc/Workbench/ReleaseNotes/ReleaseNotesWorkbench.7.48.0.Final/properties-panel-show-more-properties-for-dmn-ds-con.adoc

[id='properties-panel-show-more-properties-for-dmn-ds-con']

> = Properties panel show more properties for DMN decision services

Reviewer: ```suggestion
= Properties panel displays more properties for DMN decision services
```

> You can now visualize extra details about DMN decision services at the DMN designer properties panel in {CENTRAL}. When a DS node is selected, it's easier now to invoke it by checking the input data, encapsulated decisions, and output decisions in the properties panel.

Reviewer: ```suggestion
You can now view additional properties of DMN decision services in the DMN designer *Properties* panel in {CENTRAL}. When a decision service node is selected, you can now invoke it by checking the input data, encapsulated decisions, and output decisions in the *Properties* panel.
```
